Lib/test/test_tokenize.py

```diff
@@ -1849,8 +1849,8 @@ def test_random_files(self):
         # pass the '-ucpu' option to process the full directory.
 
         import glob, random
-        fn = support.findfile("tokenize_tests.txt")
-        tempdir = os.path.dirname(fn) or os.curdir
+        fn = support.findfile("tokenize_tests.txt", subdir='tokenizedata')
+        tempdir = os.path.dirname(os.path.dirname(fn)) or os.curdir
```

Comment:
Well, your PR *does* modify the code. So I suggest simpler code instead:

```suggestion
        tempdir = os.path.dirname(__file__) or os.curdir
```